example/rocketmq_v1alpha1_rocketmq_cluster.yaml

```diff
@@ -163,6 +163,28 @@ spec:
         spec:
           containers:
             - name: console
+              env:
+                - name: ROCKETMQ_CONFIG_LOGINREQUIRED
+                  value: "true"
```

Comment:
If login is required by default, you'd better also update the document.

Reply:
Login is not need required by default, it should be just a guide for user, i would fix it.